packages/graph/src/Graph.ts

```diff
@@ -950,6 +930,10 @@ export class Graph extends SVGZoomWidget {
     tooltip;
     tooltipHTML: (_) => string;
     tooltipFormat: (_) => string;
+
+    //  Progess Events ---
+    progress(what: "start" | "end" | "layout-start" | "layout-tick" | "layout-end") {
+    }
```

Comment:
Is the idea to override this method when you'd want to gauge performance?

Reply:
You would add an `on("progess", what=>{})` method to the hosting app.